### PR TITLE
Port 6653 Mini Guide: Connecting Cloud Resources to Services Using Tags and JQ Mapping

### PR DESCRIPTION
**Pull request by phalbert**
# Description

This guide provides step-by-step instructions on how to connect cloud resources to one or multiple services by leveraging the resource's "tags" attribute and JQ mapping techniques.

## Added docs pages

Please also include the path for the added docs

- 📝 Guides & tutorials > Connect Cloud Resources to Services (`/guides-and-tutorials/connect-cloud-resource-to-service`)

### Assets

- `static/img/guides/connectCloudServiceBlueprints.png`
- `static/img/guides/connectCloudServiceFinal.png`
- `static/img/guides/connectCloudServiceRelation.png`
- `static/img/guides/connectCloudServiceTagging.png`

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md [hunk 1]
```diff
@@ -4809,4 +4809,211 @@ In this step-by-step example, you will export your EC2 `Auto scaling groups` to
...
 
-Done! soon, you will be able to see any `Auto scaling groups`
\ No newline at end of file
+Done! soon, you will be able to see any `Auto scaling groups`
+
+
+## Connect Cloud Resources to Services.
+
+
+This guide demonstrates how to connect a Cloud Resource to a service or list of services by relying on the tags property and using JQ mapping. We will be using an S3 bucket as the Cloud Resource.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md:4818]: The guide should mention that having the S3 bucket entities already ingested by the AWS exporter is required.

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md [hunk 2]
```diff
@@ -4809,4 +4809,211 @@ In this step-by-step example, you will export your EC2 `Auto scaling groups` to
...
+
+Refer to this [AWS guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/tagging-managing.html) for more details on tagging resources in S3.
+
+
+<img src='/img/guides/connectCloudServiceTagging.png' width="75%" />
+
+<br />
+<br />
+
+2. Go to the `template-assets/aws/s3_bucket` folder from the [ingestion step](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/Installation). We are going to add a relation from our S3 bucket configuration to the `Service` blueprint by editing two files:
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md:4906]: Why use a complete URL to the docs, and not a relative URL?

Also, this link just sends the user to the installation page, without any specific part of the page that indicates where the user should go.

Also you can link to the template-assets repository directly.

Just writing `template-assets/aws/s3_buckets` does not mean anything to a reader of the docs

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please take a look at my comments

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md (hunk 2 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples.md]: Alright. I have revised the section to make it clear on what the folder represents.

**Pull request merged**